src/diffusers/loaders/lora_conversion_utils.py

```diff
@@ -663,3 +663,210 @@ def handle_qkv(sds_sd, ait_sd, sds_key, ait_keys, dims=None):
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_A.weight"] = torch.cat(
+                [lora_weight]
+            )
+
+        elif "linear1.lora_B.weight" in key:
+            q, k, v, mlp = torch.split(
+                original_state_dict.pop(f"single_blocks.{block_index}.linear1.lora_B.weight"), split_size, dim=0
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_B.weight"] = torch.cat([q])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_B.weight"] = torch.cat([k])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_B.weight"] = torch.cat([v])
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_B.weight"] = torch.cat([mlp])
+
+        elif "linear1.lora_B.bias" in key:
+            q_bias, k_bias, v_bias, mlp_bias = torch.split(
+                original_state_dict.pop(f"single_blocks.{block_index}.linear1.lora_B.bias"), split_size, dim=0
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_q.lora_B.bias"] = torch.cat(
+                [q_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_k.lora_B.bias"] = torch.cat(
+                [k_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.attn.to_v.lora_B.bias"] = torch.cat(
+                [v_bias]
+            )
+            converted_state_dict[f"single_transformer_blocks.{block_index}.proj_mlp.lora_B.bias"] = torch.cat(
+                [mlp_bias]
+            )
+
+        else:
+            new_key = key.replace("modulation.lin", "norm.linear")
+            new_key = new_key.replace("linear2", "proj_out")
+            # new_key = new_key.replace("norm.query_norm.scale", "attn.norm_q.weight")
+            # new_key = new_key.replace("norm.key_norm.scale", "attn.norm_k.weight")
+            converted_state_dict[new_key] = original_state_dict.pop(key)
+
+    def remap_final_layer(key, converted_state_dict, original_state_dict):
+        new_key = key.replace("final_layer.linear", "proj_out")
+        new_key = new_key.replace("final_layer.adaLN_modulation.1", "norm_out.linear")
+        converted_state_dict[new_key] = swap_scale_shift(original_state_dict.pop(key))
```

Comment:
I see two different keys ("final_layer.linear" and "final_layer.adaLN_modulation.1") being replaced and assigned to a `new_key`. Is this expected? 

In the conversion script we handled it differently:
https://github.com/huggingface/diffusers/blob/b66e691058652815b7791fea1c4aecb8cba26dd4/scripts/convert_flux_control_lora_to_diffusers.py#L312